sxt/cbindings/base/curve_id.h

```diff
@@ -0,0 +1,33 @@
+/** Proofs GPU - Space and Time's cryptographic proof algorithms on the CPU and GPU.
+ *
+ * Copyright 2024-present Space and Time Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace sxt::cbnb {
+//--------------------------------------------------------------------------------------------------
+// curve_id_t
+//--------------------------------------------------------------------------------------------------
+/**
+ * Ids for the various curves we support.
+ *
+ * Note: The values should match those in blitzar_api.h.
+ */
+enum class curve_id_t : unsigned {
+  curve21 = 0,
+  bls381 = 1,
+  bnp = 2,
```

Comment:
Is there any issue making these `curve25519`, `bls12_381`, and `bn254`?

Perhaps this will just be an internal tool, but `bls12_381` and `bn254` would match [Icicle](https://github.com/ingonyama-zk/icicle/blob/main/icicle/include/curves/curve_config.cuh), [Tachnyon](https://github.com/kroma-network/tachyon/tree/c02d7b6dee4856a76e4024025978778eb3838fb3/tachyon/c/math/elliptic_curves), and the Arkworks crates.

Reply:
done